### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by mikimik01**
clickup: Navigation Bar

### CODE REVIEW

File: app/src/main/java/com/example/footballmanager/ui/screens/AccountScreen.kt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.example.footballmanager.ui.screens
+
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+
+@Composable
```
Comment (PawelZ4krzewski) [inline app/src/main/java/com/example/footballmanager/ui/screens/AccountScreen.kt:7]: It's overkill to wrap Text in the other function

File: app/src/main/java/com/example/footballmanager/ui/screens/CompetitionScreen.kt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.example.footballmanager.ui.screens
+
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+
+@Composable
+fun CompetitionScreen(){
```
Comment (PawelZ4krzewski) [inline app/src/main/java/com/example/footballmanager/ui/screens/CompetitionScreen.kt:8]: same like above

File: app/src/main/java/com/example/footballmanager/ui/screens/NewsScreen.kt [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package com.example.footballmanager.ui.screens
+
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+
+@Composable
+fun NewsScreen(){
```
Comment (PawelZ4krzewski) [inline app/src/main/java/com/example/footballmanager/ui/screens/NewsScreen.kt:8]: same

File: app/src/main/java/com/example/footballmanager/ui/screens/HomeScreen.kt [hunk 4]
```diff
@@ -1,12 +1,15 @@
...
-import com.example.footballmanager.screens.result_screens.LoadingScreen
-import com.example.footballmanager.screens.result_screens.ErrorScreen
-import com.example.footballmanager.screens.result_screens.SuccessScreen
-import com.example.footballmanager.screens.view_models.RetrievingDataState
+import com.example.footballmanager.ui.RetrievingDataState
+import com.example.footballmanager.ui.screens.result_screens.LoadingScreen
+import com.example.footballmanager.ui.screens.result_screens.ErrorScreen
+import com.example.footballmanager.ui.screens.result_screens.SuccessScreen
 
+/*@NavDestination(
```
Comment (klaudiakow) [inline app/src/main/java/com/example/footballmanager/ui/screens/HomeScreen.kt]: Is this still needed?

File: app/src/main/java/com/example/footballmanager/ui/MasterLayout.kt [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+        masterViewModel.getFixturesData(ctx = ctx)
+        masterViewModel.getFixturesLiveNow()
+    }
+
+    Scaffold(modifier = Modifier.nestedScroll(scrollBehavior.nestedScrollConnection),
+        topBar = { AppTopBar(scrollBehavior = scrollBehavior) },
+        bottomBar = {
+            ButtonNavigationBar(
+                onNavigateToScreen = { screen ->
+                    navController.navigate(screen.name); masterViewModel.currentBotNavSelection =
```
Comment (klaudiakow) [inline app/src/main/java/com/example/footballmanager/ui/MasterLayout.kt]: what is this `;` :fearful:
I'd just move `masterViewModel.currentBotNavSelection =` to the next line

File: app/src/main/java/com/example/footballmanager/ui/MasterLayout.kt [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+                }
+            )
+        }
+    ) { innerPadding ->
+        Column(
+            modifier = Modifier
+                .padding(innerPadding)
+                .fillMaxSize()
+        ) {
+            NavHost(
```
Comment (klaudiakow) [inline app/src/main/java/com/example/footballmanager/ui/MasterLayout.kt]: I'd consider move it to a separate composable

File: app/src/main/java/com/example/footballmanager/ui/screens/AccountScreen.kt (hunk 1 above)
Comment (klaudiakow) [reply app/src/main/java/com/example/footballmanager/ui/screens/AccountScreen.kt:7]: @PawelZ4krzewski It's just a screen mockup

File: app/src/main/java/com/example/footballmanager/ui/theme/navigation/BottomNavigationBar.kt [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+fun NavItemIcon(modifier: Modifier = Modifier, imgVec: ImageVector) {
+    Icon(
+        imageVector = imgVec,
+        modifier = modifier.size(dimensionResource(id = R.dimen.bot_nav_bar_item_size)),
+        contentDescription = null
+    )
+}
+
+@Composable
+fun NavItemText(name: String) {
```
Comment (klaudiakow) [inline app/src/main/java/com/example/footballmanager/ui/theme/navigation/BottomNavigationBar.kt]: You don't need to isolate each component in a separate composable, especially if they are as simple as this Text

File: app/src/main/java/com/example/footballmanager/ui/screens/AccountScreen.kt (hunk 1 above)
Comment (klaudiakow) [reply app/src/main/java/com/example/footballmanager/ui/screens/AccountScreen.kt:7]: @PawelZ4krzewski It's just a screen mockup

**Review by klaudiakow — APPROVED** [review]

**Pull request merged**